scripts/globals/apkallu.lua

```diff
@@ -0,0 +1,143 @@
+xi = xi or {}
+xi.apkallu = xi.apkallu or {}
+xi.apkallu.zones = xi.apkallu.zones or {}
```

Comment:
this last one goes unused, no?

Reply:
It does get used. I just made a change to the code some but it is used to track the Apkallu mobs per zone.